examples/multiple-apps/build/index.html

```diff
@@ -12,9 +12,5 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.3/lodash.min.js"></script>
```

Comment:
if this file is generated by Webpack, let's delete it from the repo, and also ignore it from `.gitignore`.